modeshape-parent/pom.xml

```diff
@@ -188,7 +188,9 @@
         <version.postgresql.9>9.2-1002.jdbc4</version.postgresql.9>
         <version.postgresql.8>8.4-702.jdbc4</version.postgresql.8>
         <version.mysql>5.1.38</version.mysql>
+        <version.mysql.server>5.7</version.mysql.server>
         <version.com.oracle>11.2.0.4</version.com.oracle>
+        <version.oracle.it.driver>11.2.0.1.0</version.oracle.it.driver>
```

Comment:
this should be merged with the previous property. Both denote the version of the oracle driver, so we really just want to use 1.